test/jdk/TEST.groups

```diff
@@ -199,7 +199,7 @@ jdk_rmi = \
     sun/rmi
 
 jdk_security1 = \
-    java/security
+    sun/security/pkcs11
```

Comment:
Please discard this, this is just for quick testing, it's not meant to be part of this change...

Reply:
Well spotted.